### ISSUE TITLE
Live reloading is broken - can't find livereload.js

### ISSUE THREAD

**Issue by jtrussell** [jtrussell/buttle#15]
> Error: ENOENT: no such file or directory, open '/.../buttle/node_modules/tiny-lr/node_modules/livereload-js/dist/livereload.js'

**Comment by lavabear** [comment]
If anyone else is having this issue there is a workaround

 ```bash
npm install -g tiny-lr # Install  tiny-lr

cd /path/to/lib/node_modules/tiny-lr && pwd # Find path to tiny-lr

cd ../buttle/node_modules/ && rm -rf tiny-lr # Remove buttle tiny-lr

ln -s /path/to/tiny-lr tiny-lr #Add symbolic link to tiny-lr 
```

**Comment by jtrussell** [comment]
Thanks! Is this an issue with the version of tiny-lr used by buttle?

**Comment by jtrussell** [comment]
I think this may have also been fixed with 0.3.1.
